main.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   naming_suffix                = "external-tableau-${var.naming_suffix}"
   naming_suffix_2018_vanilla   = "ext-tableau-2018-vanilla-${var.naming_suffix}"
+  naming_suffix_linux          = "ext_tableau_linux-${var.naming_suffix}"
```

Comment:
Think these should be hyphens instead of underscores

Reply:
Fixed in commit ae1da21.